js/modules/loki_message_api.js

```diff
@@ -112,7 +112,7 @@ class LokiMessageAPI {
     }
 
     const params = {
-      pubKey,
+      pubKey: window.getEnvironment() === 'production' ? pubKey : pubKey.substring(0, pubKey.length - 2),
```

Comment:
Can you remind me what is the difference between production and development pubkeys?

Reply:
The plan for now is to just rip the last 2 characters off the end of the pubkey that is sent to the storage server
This won't affect anything besides the retrieve/store and pubkey -> swarm mapping (which I'm not fussed about on testnet)
I.e. the actual pubkey used for encryption/decryption will still be untouched inside the envelope
The reason for this is currently the only "validity" check being done on the pubkeys storage server side is that they are length 66, so to ensure that testnet pubkeys get rejected by mainnet nodes and vice versa we wanted to change the length of the keys

Reply:
Ok, thanks for the update !